Bigtable/src/Instance.php

```diff
@@ -202,57 +214,57 @@ public function state(array $options = [])
     }
 
     /**
-     * Create a new instance.
-     *
      * Example:
      * ```
-     * $bigtable = new Google\Cloud\Bigtable\BigtableClient();
+     * use Google\Cloud\Bigtable\BigtableClient;
+     * $bigtable = new BigtableClient();
      * $operation = $instance->create(
-     *     $bigtable->clusterMetadata(['clusterId' => 'my-cluster', 'locationId' => 'us-east1-c'])
+     *     [$bigtable->buildClusterMetadata('my-cluster', 'my-location', null, 3)]
      * );
      * ```
      *
      * @codingStandardsIgnoreStart
-     * @see https://cloud.google.com/bigtable/docs/reference/admin/rpc/google.bigtable.admin.v2#CreateInstanceRequest CreateInstanceRequest
+     * @see https://cloud.google.com/bigtable/docs/reference/admin/rpc/
+     *     google.bigtable.admin.v2#CreateInstanceRequest CreateInstanceRequest
      * @codingStandardsIgnoreEnd
      *
-     * @param array $clusterMetadataList [
-     *        $bigtable->clusterMetadata('clusterId1', 'locationId1', 'SSD', 3),
-     *        $bigtable->clusterMetadata('clusterId2', 'locationId2', 'SSD', 3)
-     * ]
-     * @param string $displayName **Defaults to** the value of $instanceId.
-     * @param array $labels as key/value pair ['foo' => 'bar']. For more information, see
-     *        [Using labels to organize Google Cloud Platform resources](https://cloudplatform.googleblog.com/2015/10/using-labels-to-organize-Google-Cloud-Platform-resources.html).
-     * @param int $type Possible values are represented by the following constants:
-     *        `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_PRODUCTION`,
-     *        `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_DEVELOPMENT` and
-     *        `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_UNSPECIFIED`.
-     *        **Defaults to** using `Google\Cloud\Bigtable\Instance::INSTANCE_TYPE_UNSPECIFIED`.
-     *
+     * @param array[] $clusterMetadataList Use {@see Google\Cloud\Bigtable\BigtableClient::buildClusterMetadata()}
+     *        to create properly formatted cluster configurations.
+     *        API accepts a map where the cluster ID is the php array key.
```

Comment:
I don't want this comment here. I want it inline, directly above where I commented. Here it only would add confusion because `$clusterMetadataList` is not associative.